lib/src/json_rpc/methods.rs

```diff
@@ -540,6 +540,33 @@ define_methods! {
     sudo_networkState_event(subscription: Cow<'a, str>, result: NetworkEvent) -> (),
 }
 
+#[derive(Debug, Clone)]
+pub struct MerkleProof(pub Vec<u8>);
```

Comment:
```suggestion
pub struct MerkleProof(pub Vec<HexString>);
```

I think that putting the decoded proof here makes more sense, this way you're guaranteed that serializing can't fail.